test/d2i_test.c

```diff
@@ -40,13 +40,13 @@ static int execute_test(D2I_TEST_FIXTURE fixture)
 {
     BIO *bio = NULL;
     ASN1_VALUE *value = NULL;
-    int ret = 1;
+    int ret = 0;
     unsigned char buf[2048];
     const unsigned char *buf_ptr = buf;
     int len;
 
     if ((bio = BIO_new_file(test_file, "r")) == NULL)
-        return 1;
+        return 0;
```

Comment:
Yeah, but it's not consistent and I thought that was also an issue.  As long as doing that is a conscious deciosion, I'm okay with this.
+1